src/sidebar/components/ThreadCard.tsx

```diff
@@ -63,22 +61,29 @@ function ThreadCard({ frameSync, thread }: ThreadCardProps) {
   }, [focusRequest, store, thread.id]);
 
   return (
-    /* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
     <Card
       active={isHovered}
       classes="cursor-pointer focus-visible-ring theme-clean:border-none"
       data-testid="thread-card"
       elementRef={cardRef}
-      tabIndex={-1}
+      tabIndex={0}
+      role="button"
+      aria-label="Press Enter to scroll annotation into viewport"
```

Comment:
I have no idea what would be a sensible message for screen readers. I don't think non-sighted people care about the annotation being in the viewport or not 😅

Reply:
A couple of ideas:

`Press Enter to scroll annotation into view` could do the trick.

`Press Enter to scroll to annotation in document` maybe be awkward when paired with video transcripts and later, image annotations. 

So I'd vote for removing the "port" from "viewport" at this time.